atproto/lexicon/cmd/lextool/net.go

```
@@ -54,17 +53,24 @@ func runValidateRecord(cctx *cli.Context) error {
 		pdsURL, ident.DID, aturi.Collection(), aturi.RecordKey())
 	resp, err := http.Get(url)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "in http.Get")
 	}
+	defer resp.Body.Close()
+
 	if resp.StatusCode != http.StatusOK {
 		return fmt.Errorf("fetch failed")
 	}
+
 	respBytes, err := io.ReadAll(resp.Body)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "in ReadAll")
 	}
 
 	body, err := data.UnmarshalJSON(respBytes)
+	if err != nil {
```

This error was not being checked.